launch-game.sh

```diff
@@ -25,7 +25,12 @@ mono OpenRA.Game.exe Engine.LaunchPath="$MODLAUNCHER" $MODARG "$@"
 
 # Show a crash dialog if something went wrong
 if [ $? != 0 ] && [ $? != 1 ]; then
-	ERROR_MESSAGE="OpenRA has encountered a fatal error.\nPlease refer to the crash logs and FAQ for more information.\n\nLog files are located in ~/.openra/Logs\nThe FAQ is available at http://wiki.openra.net/FAQ"
+	LOGS="~/.openra/Logs"
```

Comment:
It would be nice if we could show the correct path for macOS as part of this fix too.

Something like (not tested):
```bash
if [[ "$OSTYPE" == "darwin"* ]]; then
	LOGS="~/Library/Application Support/OpenRA/Logs/"
else
	LOGS="~/.openra/Logs"
	if [ ! -d "$LOGS" ]; then
		LOGS="~/.config/openra/Logs"
	fi
fi
```

Reply:
Done.